README.md

```diff
@@ -27,7 +27,7 @@ technology stack built on the following principles:
   without extra boilerplate code using DSL based on arrays.
 
   ```javascript
-  mt.case({
+  mt.case('Test utils.duration', { utils }, {
```

Comment:
Maybe replace `utils` with `common` since this example is based on the test from there.